armi/utils/asciimaps.py

```diff
@@ -180,22 +183,44 @@ def _getIndices(base, col):  # pylint: disable=no-self-use
         return iBase + col * 2, jBase - col
 
     def writeMap(self, stream):
-        """Writing this is easiest if we just make a big ascii map and fill it in."""
+        """Write the lattice to a text stream"""
         grid = {}
+        # The length of the largest string contents. Useful for spacing everything out
+        # nicely
+        maxSize = len(str(next(iter(self.lattice.values()))))
```

Comment:
groan is that really what we have to do to get the length of the first value!? Python 3 whyyyy

Reply:
Yeah, I had a private little hissy of my own when I found out that the various dict view classes arent _themselves_ iterators. It feels like the `iter()` call should be unnecessary.